mapie/tests/test_subsample.py

```diff
@@ -32,6 +32,31 @@ def test_split_SubSample() -> None:
     np.testing.assert_equal(tests, tests_expected)
 
 
+def test_split_SubSample_n_samples() -> None:
```

Comment:
```suggestion
@pytest.mark.parametrize("n_samples", [0.8, 6])
def test_split_SubSample_n_samples(n_samples: Union[int, float]) -> None:
```
Prefer to use this convention when testing several values of a function.

Reply:
However, I like your test but it has the wrong name. Suggest a test called `test_n_samples_equivalence' which tests that using `n_samples` and `n_ratio=n_samples/n_all` produces the same split.